target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4029-insect-common.dtsi

```diff
@@ -282,8 +307,8 @@
 	wifi@0,0 {
 		compatible = "qcom,ath10k";
 		reg = <0x00010000 0 0 0 0>;
-		nvmem-cells = <&mac_address 1>;
-		nvmem-cell-names = "mac-address";
+		nvmem-cells = <&cal_factory_9000>;
```

Comment:
If I'm not mistaken, the MAC address-related NVMEM cells should stay together with new caldata, ART blobs do not contain the MACs on insects-s